yellowbrick/target/binning.py

```diff
@@ -181,6 +186,10 @@ def balanced_binning_reference(y, ax=None, target="y", bins=4, **kwargs):
 
     # Fit and show the visualizer
     visualizer.fit(y)
-    visualizer.show()
+
+    if show:
+        visualizer.show()
+    else:
+        visualizer.finalize()
```

Comment:
I think this is what was missing! We have the `show=False` parameter set on @Kautumn06's quick method test (as it should be!), but because we hadn't yet added the show/finalize logic to the actual quick method yet, when we got to the Appeyor/Miniconda test, it was trying to `finalize` instead of `show`, causing the test to hang indefinitely. This should fix it 🤞 🙏 😅